### PR TITLE
31 - Updated querying tutorial with words instead of letters in examples

### PR DESCRIPTION
**Pull request by deakgeek**
#31

### CODE REVIEW

**Comment by arynhuck** [comment]
Merged master in and fixed conflicts @helmoski

File: content/efcore/getting-started/querying.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ Loading all of the entities from a database is useful, but there are many use ca
 using (var context = new LibraryContext()) 
 { 
     var books = context.Books 
-        .Where(b => b.Genre == "Historical") 
+        .Where(b => b.Genre == "Historical")
```
Comment (helmoski) [inline content/efcore/getting-started/querying.md]: This should be `.Where(book => book.Genre == "Historical")` right?

**Review by helmoski — CHANGES_REQUESTED** [review]

File: content/efcore/getting-started/querying.md (hunk 1 above)
Comment (arynhuck) [reply content/efcore/getting-started/querying.md]: Yeah, it looks like I messed up and accidentally deleted the wrong thing when resolving the conflicts. Give me a sec.

**Comment by helmoski** [comment]
So, C# doesn't allow the variable in the LINQ statement to share the same name as an outer variable. Closing this PR/branch because we will stick with using the letter instead.

**Pull request closed without merging**